screens/LuresResults.js

```diff
@@ -960,32 +978,193 @@ export default function LuresResults({ route }) {
+    {reactIfView(currentTutorial == 'MoonPhase', <Tooltip
+      contentStyle={tutorial_styles.weatherAndMoonLine}
+      backgroundColor={tutorial_transparent}
+      isVisible={currentTutorial == 'MoonPhase'}
+      content={<Text style={[text_style.fontColorWhite]}>{loadTranslations("tutMoonPhase")}</Text>}
+      placement="top"
+      onClose={async () => {setCurrentTutorial(await updateTutorialAndGetNext('MoonPhase', NAV_LURES_RESULTS))}}
+    >
+      <TooltipChildrenContext.Consumer>
+        {({ tooltipDuplicate }) => (
+          reactIfView(!tooltipDuplicate,
+            <View style={[{height:Platform.OS === 'android' ? 50 : 0, width: width}]}></View>
+          )
+        )}
+      </TooltipChildrenContext.Consumer>
+    </Tooltip>
+    )}
+    {reactIfView(currentTutorial == 'PreviousWeather', <Tooltip
+      contentStyle={tutorial_styles.weatherAndMoonLine}
+      backgroundColor={tutorial_transparent}
+      isVisible={currentTutorial == 'PreviousWeather'}
+      content={<Text style={[text_style.fontColorWhite]}>{loadTranslations("tutPreviousWeather")}</Text>}
+      placement="top"
+      onClose={async () => {setCurrentTutorial(await updateTutorialAndGetNext('PreviousWeather', NAV_LURES_RESULTS))}}
+    >
+      <TooltipChildrenContext.Consumer>
+        {({ tooltipDuplicate }) => (
+          reactIfView(!tooltipDuplicate,
+            <View style={[{height:Platform.OS === 'android' ? 50 : 0, width: width}]}></View>
+          )
+        )}
+      </TooltipChildrenContext.Consumer>
+    </Tooltip>
+    )}
     <View style={{ flexDirection: 'row', justifyContent: 'space-between', padding: 10, alignItems: 'center' }}>
+      
       <View style={{ flexDirection: 'row', alignItems: 'center' }}>
-      <MaterialIcons name="dark-mode" size={22} color="black" />
-      <Text style={{ marginLeft: 5 }}>{moonPhaseLabels[moonPhase.prevPhase.phase]}</Text>
-    </View>
+        <MaterialIcons name="dark-mode" size={22} color="black" />
+        <Text style={{ marginLeft: 5 }}>{moonPhaseLabels[moonPhase.prevPhase.phase]}</Text>
+      </View>
       <View style={{ flexDirection: 'row', alignItems: 'center' }}>
-      <MaterialIcons name="water-drop" size={24} color="black" />
-      <Text style={{ marginLeft: 5 }}>{weather.precipitation ? 'Yes' : 'No'}</Text>
-    </View>
-    <View style={{ flexDirection: 'row', alignItems: 'center' }}>
-      <MaterialIcons name="air" size={24} color="black" />
-      <Text style={{ marginLeft: 5 }}>{weather.windGust} mph</Text>
-    </View>
+        <MaterialIcons name="water-drop" size={24} color="black" />
+        <Text style={{ marginLeft: 5 }}>{weather.precipitation ? 'Yes' : 'No'}</Text>
+      </View>
+      <View style={{ flexDirection: 'row', alignItems: 'center' }}>
+        <MaterialIcons name="air" size={24} color="black" />
+        <Text style={{ marginLeft: 5 }}>{weather.windGust} mph</Text>
+      </View>
     
-  </View>
-  );
+    </View>
+  </>);
```

Comment:
Is this a fragment?

Reply:
Yes, because the Tooltip has to appear above the element itself, and putting it in the view would create some placement issues. I'll remove this once conditions-results branch is merged into this branch.